assignment-day-2/src/pipeline/utils.py

```diff
@@ -0,0 +1,10 @@
+import psycopg2
+
+def connect():
+    return psycopg2.connect(
+        host = 'localhost',
+        database = 'data-internship',
+        user='postgres',
+        password='nischal541',
+        port = 5432
+    )
```

Comment:
```suggestion
import psycopg2

def connect():
    return psycopg2.connect(
        host = 'localhost',
        database = 'data-internship',
        user='postgres',
        password='nischal541',
        port = 5432
    )
    
``` 
Database secrets are sensitive and should not be displayed directly. How about we add these to an env variable and then use them?